src/en/superstream/src/eu/kanade/tachiyomi/animeextension/en/superstream/SuperStreamAPI.kt

```diff
@@ -0,0 +1,401 @@
+            val data = (queryApiParsed<SeriesDataProp>(apiQuery)).data
+                ?: throw RuntimeException("API error")
+
+            val episodes = data.season.mapNotNull {
+                val seasonQuery =
+                    """{"childmode":"$hideNsfw","app_version":"11.5","year":"0","appid":"$appId","module":"TV_episode","display_all":"1","channel":"Website","season":"$it","lang":"en","expired_date":"${getExpiryDate()}","platform":"android","tid":"${loadData.id}"}"""
+                (queryApiParsed<SeriesSeasonProp>(seasonQuery)).data
+            }.flatten().reversed()
+
+            return Pair(null, Pair(data, episodes))
+        }
+    }
+
+    fun loadLinks(data: String): List<Video> {
+        val videoList = mutableListOf<Video>()
+        val subsList = mutableListOf<Track>()
+
+        val parsed = parseJson<LinkData>(data)
+
+        // No childmode when getting links
+        val query = if (parsed.type == TYPE_MOVIES) {
+            """{"childmode":"0","uid":"","app_version":"11.5","appid":"$appId","module":"Movie_downloadurl_v3","channel":"Website","mid":"${parsed.id}","lang":"","expired_date":"${getExpiryDate()}","platform":"android","oss":"1","group":""}"""
+        } else {
+            val episode = parsed.episode ?: throw RuntimeException("No episode number!")
+            val season = parsed.season ?: throw RuntimeException("No season number!")
+            """{"childmode":"0","app_version":"11.5","module":"TV_downloadurl_v3","channel":"Website","episode":"$episode","expired_date":"${getExpiryDate()}","platform":"android","tid":"${parsed.id}","oss":"1","uid":"","appid":"$appId","season":"$season","lang":"en","group":""}"""
+        }
+
+        val linkData = queryApiParsed<LinkDataProp>(query)
+
+        if (linkData.code == -102) {
+            return arrayListOf()
+        }
+
+        // Should really run this query for every link :(
+        val fid = linkData.data[0]!!.list.firstOrNull { it.fid != null }?.fid
+
+        val subtitleQuery = if (parsed.type == TYPE_MOVIES) {
+            """{"childmode":"0","fid":"$fid","uid":"","app_version":"11.5","appid":"$appId","module":"Movie_srt_list_v2","channel":"Website","mid":"${parsed.id}","lang":"en","expired_date":"${getExpiryDate()}","platform":"android"}"""
+        } else {
+            """{"childmode":"0","fid":"$fid","app_version":"11.5","module":"TV_srt_list_v2","channel":"Website","episode":"${parsed.episode}","expired_date":"${getExpiryDate()}","platform":"android","tid":"${parsed.id}","uid":"","appid":"$appId","season":"${parsed.season}","lang":"en"}"""
+        }
+
+        val subtitles = queryApiParsed<SubtitleDataProp>(subtitleQuery).data
+        subtitles?.list?.forEach {
+            it.subtitles.forEach second@{ sub ->
+                if (sub.filePath.isNullOrBlank().not()) {
+                    subsList.add(
+                        Track(
```

Comment:
This needs to be wrapped in a try block and catch any `Error`s because the current stable version doesn't have the class `Track`.

See how it's done in other extensions like Vidembed and test it out on the stable version.